### PR TITLE
Add timestamped email to prove Google was using those DKIM keys

### PR DESCRIPTION
**Pull request by petertodd**
FWIW I have permission from @gmaxwell to publish this email.

### CODE REVIEW

**Comment by gmaxwell** [comment]
ACK, permission. :)

File: README.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ provide the correct public-key.
 
 ### "What about this page that says DKIM can be fooled?"
 
-I lot of confident Twitter "experts" proclaim tha DKIM can be faked.
+A lot of confident Twitter "experts" proclaim tha DKIM can be faked.
```
Comment (Sjors) [inline README.md:160]: Nit: `tha` typo

Comment (petertodd) [reply README.md:160]: ?

That's from another pull req fixing the typo: https://github.com/robertdavidgraham/hunter-dkim/pull/3

**Comment by Sjors** [comment]
Tested e15ae627284a3511214191f6af0f599116eb6a27 on Ubuntu. It considers this email valid and verifies the timestamp.

**Comment by gmaxwell** [comment]
@Sjors You might want to drop the --no-bitcoin in your copy of the validation script, since I know you're in a position to also check against a local copy of Bitcoin-- that will give a more vigorous third party validation of the timestamp.

**Comment by Sjors** [comment]
@gmaxwell I used `ots verify timestamped.eml.ots`, which breaks when I turn off my node.

**Review by ValZapod — APPROVED** [review]

**Pull request merged**